### PR TITLE
MAGEDOC-3489 Cron logging updates

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- Content update

## Summary

When this pull request is merged, it will...

Add info about the new cron.log workflow.

Fixes #MAGEDOC-3489 

whatsnew
Added information about the new `cron.log` file to [Logging](https://devdocs.magento.com/guides/v2.3/config-guide/cli/logging.html) and [Configure and run cron](https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-cron.html).

### CODE REVIEW

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md [hunk 1]
```diff
@@ -39,46 +38,48 @@ This command has no effect on cron jobs outside the `#~ MAGENTO START` and `#~ M
...
-System logs provide detailed information for debugging. The following attributes apply to Magento cron logs:
+With Magento 2.3.1, `cron` job information is moved from `system.log` and into a separate `cron.log`.
+By default, the cron information can be found at `<install_directory>/var/log/cron.log`. 
+All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
 
--   All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
+In addition to being logged in `cron.log`:
 
--   Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<your Magento install dir>/var/log` directory.
+- Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<install_directory>/var/log` directory.
```
Comment (keharper) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: In which specific file(s) do these entries appear?

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md [hunk 2]
```diff
@@ -39,46 +38,48 @@ This command has no effect on cron jobs outside the `#~ MAGENTO START` and `#~ M
...
 
--   Jobs with an `ERROR` status are always logged as `CRITICAL` in `<your Magento install dir>/var/log/exception.log`.
+- Jobs with an `ERROR` status are always logged as `CRITICAL` in `<install_directory>/var/log/exception.log`.
 
--   Jobs with a `MISSED` status are logged as `INFO` in the `<your Magento install dir>/var/log` directory (developer mode only).
+- Jobs with a `MISSED` status are logged as `INFO` in the `<install_directory>/var/log` directory (developer mode only).
```
Comment (keharper) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: Consider putting these 3 bullet items into a table (not sure if there's enough info at the moment)

File: guides/v2.3/config-guide/cli/logging.md [hunk 3]
```diff
@@ -67,6 +67,14 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
     bin/magento cache:flush
     ```
 
+## Cron logging
+
+With the release of version 2.3.1, Magento now creates a separate `cron` log. 
+Magento recently made cron logging more verbose, which provided more information but lengthened the `system.log` considerably. 
+Moving `cron` info to a dedicated log will make both logs easier to read.
```
Comment (keharper) [inline guides/v2.3/config-guide/cli/logging.md]:
```suggestion
Moving `cron` info to a dedicated log makes both logs easier to read.
```

**Comment by keharper** [comment]
2.2 PR: https://github.com/magento/magento2/pull/18389

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md [hunk 4]
```diff
@@ -29,8 +29,7 @@ To remove the Magento crontab:
 1.  Log in as or switch to the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
 2.  Change to the Magento installation directory.
 3.  Enter the following command:
-
-        php bin/magento cron:remove
+      `php bin/magento cron:remove`
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: Done

File: guides/v2.3/config-guide/cli/logging.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Got it.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md [hunk 5]
```diff
@@ -39,46 +38,48 @@ This command has no effect on cron jobs outside the `#~ MAGENTO START` and `#~ M
...
-System logs provide detailed information for debugging. The following attributes apply to Magento cron logs:
+With Magento 2.3.1, `cron` job information was moved from `system.log` and into a separate `cron.log`.
+By default, the cron information can be found at `/var/log/cron.log`. 
+All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
 
--   All exceptions from cron jobs are logged by `\Magento\Cron\Observer\ProcessCronQueueObserver::execute`.
+In addition to being logged in `cron.log`:
 
--   Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<your Magento install dir>/var/log` directory.
+- Failed jobs with `ERROR` and `MISSED` statuses are logged to the `<your Magento install dir>/var/log` directory.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: Added them. Good one.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: Don't see it as tabular data. A bit too verbose I think.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: Fixed.

**Comment by keharper** [comment]
For some reason, GH wouldn't let me make a comment near the top of the `logging.md` file. In line 14 or 15 in each version of the file, there's a reference to "develop" mode. Change those to "developer" mode.

File: guides/v2.3/config-guide/cli/logging.md [hunk 6]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
+
 ## Syslog logging
 
 By default, Magento writes _syslog_ logs to the operating system `syslog` file.
 
 #### To enable syslog logging:
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
+As of Magento 2.3.1, enabling the syslog must be done via the Magento CLI. 
+The setting in the Admin panel has been removed.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Done.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md [hunk 7]
```diff
@@ -60,25 +59,27 @@ You must run cron twice: the first time to discover tasks to run and the second
...
--   Scheduled date
--   Executed date
--   Finished date
+- Job ID and code
+- Status
+- Created date
+- Scheduled date
+- Executed date
+- Finished date
 To see records in the table, log in to the Magento database on the command line and enter `SELECT * from cron_schedule;`.'
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-cron.md]: Thanks.

File: guides/v2.2/config-guide/cli/logging.md [hunk 8]
```diff
@@ -60,8 +60,48 @@ To enable database logging:
...
 
+1. Flush the cache.
+
+    ```bash
+    bin/magento cache:flush
+    ```
+
+## Cron logging
+
+With the release of version 2.3.1, Magento now creates a separate `cron` log.
```
[the remark this answers is not in the document]
Comment (dobooth) [reply guides/v2.2/config-guide/cli/logging.md]: Done.

File: guides/v2.2/config-guide/cli/logging.md [hunk 9]
```diff
@@ -60,8 +60,48 @@ To enable database logging:
...
+
+By default, Magento writes `cron` info to `<install_directory>/var/log/cron.log`.
+
+## Syslog logging
+
+By default, Magento writes _syslog_ logs to the operating system `syslog` file.
+
+#### To enable syslog logging:
+
+1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
```
Comment (fascinosum) [inline guides/v2.2/config-guide/cli/logging.md]: Use the `setup:config:set --enable-syslog-logging=true|false` command to change the behavior

File: guides/v2.2/config-guide/cli/logging.md [hunk 10]
```diff
@@ -60,8 +60,48 @@ To enable database logging:
...
+
+    ```bash
+    bin/magento config:set dev/syslog/syslog_logging 1
```
Comment (fascinosum) [inline guides/v2.2/config-guide/cli/logging.md]: bin/magento setup:config:set --enable-syslog-logging=true

File: guides/v2.2/config-guide/cli/logging.md [hunk 11]
```diff
@@ -60,8 +60,48 @@ To enable database logging:
...
     ```bash
     bin/magento cache:flush
     ```
+
+#### To disable syslog logging:
+
+1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `0`.
+
+    ```bash
+    bin/magento config:set dev/syslog/syslog_logging 0
```
Comment (fascinosum) [inline guides/v2.2/config-guide/cli/logging.md]: bin/magento setup:config:set --enable-syslog-logging=false

File: guides/v2.2/config-guide/cli/logging.md [hunk 12]
```diff
@@ -60,8 +60,48 @@ To enable database logging:
...
+
 2. Flush the cache.
 
     ```bash
     bin/magento cache:flush
     ```
+
+#### To disable syslog logging:
+
+1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `0`.
```
Comment (fascinosum) [inline guides/v2.2/config-guide/cli/logging.md]: Use the `setup:config:set --enable-syslog-logging=true|false` command to change the behavior

File: guides/v2.3/config-guide/cli/logging.md [hunk 13]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
+With the release of version 2.3.1, Magento now creates a separate `cron` log. 
+Magento recently made cron logging more verbose, which provided more information but lengthened the `system.log` considerably. 
+Moving `cron` info to a dedicated log makes both logs easier to read.
+
+By default, Magento writes `cron` info to `<install_directory>/var/log/cron.log`.
+
 ## Syslog logging
 
 By default, Magento writes _syslog_ logs to the operating system `syslog` file.
+As of Magento 2.3.1, you must use the `magento` command to enable or disable the syslog.
```
Comment (fascinosum) [inline guides/v2.3/config-guide/cli/logging.md]: you must use `setup:config:set --enable-syslog-logging=true|false` command

File: guides/v2.3/config-guide/cli/logging.md [hunk 14]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
+The setting in the Magento Admin has been removed.
 
 #### To enable syslog logging:
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
+Logging to `syslog` is always enabled for developer and default modes, but disabled for production mode; even on Cloud where only production mode is supported.
```
Comment (fascinosum) [inline guides/v2.3/config-guide/cli/logging.md]: Logging to `syslog` is disable by default

File: guides/v2.3/config-guide/cli/logging.md [hunk 15]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
+
+1. Use the `setup:config:set` command to change the `dev/syslog/syslog_logging` database value to `true`.
```
Comment (fascinosum) [inline guides/v2.3/config-guide/cli/logging.md]: Use the `setup:config:set --enable-syslog-logging=true|false` command to change the behavior

File: guides/v2.3/config-guide/cli/logging.md [hunk 16]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
+2. Flush the cache.
 
     ```bash
     bin/magento cache:flush
     ```
 
 #### To disable syslog logging:
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `0`.
+1. Use the `setup:config:set` command to change the `dev/syslog/syslog_logging` database value to `false`.
```
Comment (fascinosum) [inline guides/v2.3/config-guide/cli/logging.md]: Use the `setup:config:set --enable-syslog-logging=true|false` command to change the behavior

File: guides/v2.3/config-guide/cli/logging.md (hunk 13 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Fixed.

Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Fixed.

File: guides/v2.3/config-guide/cli/logging.md (hunk 14 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Fixed.

File: guides/v2.3/config-guide/cli/logging.md (hunk 16 above)
Comment (dobooth) [reply guides/v2.3/config-guide/cli/logging.md]: Fixed.

File: guides/v2.3/config-guide/cli/logging.md [hunk 17]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
 
     ```bash
     bin/magento cache:flush
     ```
 
-#### To disable syslog logging:
+### To disable syslog logging
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `0`.
+1. Use the `setup:config:set` command to change the `dev/syslog/syslog_logging` database value to `false`.
```
Comment (fascinosum) [inline guides/v2.3/config-guide/cli/logging.md]: There is no any `dev/syslog/syslog_logging` value in the database

File: guides/v2.3/config-guide/cli/logging.md [hunk 18]
```diff
@@ -67,34 +68,46 @@ By default, Magento writes database activity logs to the `var/debug/db.log` file
...
+As of Magento 2.3.1, you must use the `magento` command to enable or disable the syslog.
+The setting in the Magento Admin has been removed.
 
-#### To enable syslog logging:
+### To enable syslog logging
 
-1. Use the `config:set` command to change the `dev/syslog/syslog_logging` database value to `1`.
+Logging to `syslog` is disabled by default.
+
+1. Use the `setup:config:set` command to change the `dev/syslog/syslog_logging` database value to `true`.
```
Comment (fascinosum) [inline guides/v2.3/config-guide/cli/logging.md]: There is no any `dev/syslog/syslog_logging` value in the database

**Pull request merged**